### PR TITLE
Apply system-fetcher findings

### PR DESCRIPTION
**Pull request by kaloyanspiridonov**
**Description**

Changes proposed in this pull request:
- the system-fetcher only process tenants which are global accounts
- remove one of the sLIS API filters
- add additional query param to the sLIS request
- use `productId` instead of `ppmsProductLineOfficialName` for system filtering
- store `productId` and `ppmsProductVersionId` as application labels
- map system `mainUrl` to app `baseUrl`
- remove tenant isolation for system-fetcher

- [X] Implementation
- [X] Unit tests
- [X] Integration tests
- [X] `chart/compass/values.yaml` is updated 
- [X] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/repo/upsert.go [hunk 1]
```diff
@@ -125,33 +139,24 @@ func (u *upserter) unsafeUpsert(ctx context.Context, resourceType resource.Type,
 		return "", apperrors.NewInternalError("id cannot be empty, check if the entity implements Identifiable")
 	}
 
-	if resourceType.IsTopLevel() && id == upsertedID {
```
Comment (NickyMateev) [inline components/director/internal/repo/upsert.go:128]: Why did we have to remove this `id == upsertedID` check?

File: tests/system-fetcher/tests/handler_test.go [hunk 2]
```diff
@@ -47,6 +47,8 @@ const (
 		"systemNumber": "%d",
```
Comment (NickyMateev) [inline tests/system-fetcher/tests/handler_test.go:47]: How much effort would it be to have the e2e test which ensures that the many to many table is leveraged when one and the same app is bound to multiple tenants?

**Review by dzahariev — APPROVED** [review]

**Review by desislavaa — APPROVED** [review]

**Review by StanislavStefanov — APPROVED** [review]

**Pull request merged**